packages/ui/src/app/pages/Proposals/PastProposals.tsx

```diff
@@ -29,6 +29,12 @@ export const PastProposals = () => {
 
   const { activities } = useProposalsActivities()
 
+  const onApplyFilter = (filter: ProposalFiltersState) => {
+    console.log('Saving : ', JSON.stringify(filter))
+    setFilters(filter)
+    localStorage.setItem('lastFilter', JSON.stringify(filter))
```

Comment:
Same as above

Reply:
Hello @WRadoslaw 
I updated the code and please check it.